src/umpire/strategy/DynamicPoolMap.cpp

```diff
@@ -416,6 +432,16 @@ std::size_t DynamicPoolMap::releaseFreeBlocks()
     }
   }
 
+#if defined(UMPIRE_ENABLE_BACKTRACE)
+  if (released_bytes > 0) {
+    umpire::util::backtrace bt{};
+    umpire::util::backtracer<>::get_backtrace(bt);
+    UMPIRE_LOG(Info, "actual_size: " << m_actual_bytes 
+      << " (prev: " << (m_actual_bytes+released_bytes) 
+      << ") " << umpire::util::backtracer<>::print(bt));
+  }
+#endif
+
```

Comment:
What about DynamicPoolList?

Reply:
That's handled by the DynamicSizePool.hpp file.